community/libgdata/APKBUILD

```diff
@@ -1,36 +1,53 @@
-# Contributor: 
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=libgdata
 pkgver=0.17.9
-pkgrel=1
+pkgrel=2
 pkgdesc="access library for gdata"
-url="http://live.gnome.org/libgdata"
+url="https://wiki.gnome.org/Projects/libgdata"
 arch="all"
-license="LGPL"
-makedepends="intltool uhttpmock-dev glib-dev 
-	libsoup-dev json-glib-dev liboauth-dev"
-checkdepends="gsettings-desktop-schemas"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-source="https://download.gnome.org/sources/libgdata/${pkgver%.*}/libgdata-${pkgver}.tar.xz"
-builddir="${srcdir}/${pkgname}-${pkgver}"
+license="LGPL-2.1-or-later"
+depends="gsettings-desktop-schemas"
+makedepends="intltool uhttpmock-dev glib-dev vala gobject-introspection-dev
+	libsoup-dev json-glib-dev liboauth-dev gnome-online-accounts-dev gcr-dev
+	automake autoconf libtool"
+subpackages="$pkgname-static $pkgname-dev $pkgname-doc $pkgname-lang"
+source="https://download.gnome.org/sources/libgdata/${pkgver%.*}/libgdata-${pkgver}.tar.xz
+	only-ipv4-tests.patch"
+
+prepare() {
+	default_prepare
+
+	# Timeout on armhf
+	case "${DRONE_REPO_OWNER}-${DRONE_RUNNER_PLATFORM}-${CARCH}" in
+		alpinelinux-linux/arm-armhf)
+			sed -i -e '/perf/d' gdata/tests/Makefile.am
+		;;
+		alpinelinux-*-*)
+			echo "DroneCI: ${DRONE_REPO_OWNER}-${DRONE_RUNNER_PLATFORM}-${CARCH}"
+		;;
+	esac
+
+	intltoolize --copy --force
+	autoreconf -if
```

Comment:
```suggestion
	autoreconf --install --force
```

Reply:
...but why? Both are perfectly valid.

Reply:
The long options are more explicit. It saves reviewers having to reference the man page.

Reply:
Or a `autoreconf -h`. Anyway, I get your point, I'll do that for future PRs. (Am not at PC right now)